src/server/services/afis/afis-facturen.ts

```diff
@@ -273,28 +274,24 @@ function transformFacturen(
   };
 }
 
-function shouldFilterOutFactuur(postingDate: string): boolean {
-  const postingDateAsDate = new Date(postingDate);
+function isDownloadAvailable(postingDate: string): boolean {
+  if (!postingDate) {
+    return true;
+  }
+
+  const datePosted = parseISO(postingDate);
   const now = new Date();
 
-  if (postingDateAsDate.getTime() > now.getTime()) {
-    captureMessage('postingDate of invoice is in the future', {
-      severity: 'error',
-    });
+  if (datePosted.getTime() > now.getTime()) {
```

Comment:
Kun je dan niet beter iets doen als:

```
if (isAfter(datePosted, now)) {
  ...
}
```

Dat is wat leesbaarder toch?

Reply:
Of

```
const isDatePostedInThaFuture = datePosted.getTime() > now.getTime();
if (isDatePostedInThaFuture) {
  return false;
}
```

Reply:
Goede, ook erin gezet